services/retrieve_and_rank/v1.js

```diff
@@ -578,7 +585,7 @@ module.exports = function (RED) {
 
                 default:
                   node.status({});
-                  var message = "Cluster status: "+res.status+", description: "+res.status_description;
+                  var message = 'Cluster status: '+res.status+', description: '+res.status_description;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Variable 'message' is already declared in line 569.](https://www.codacy.com/app/BetaWorks-NodeRED-Watson/node-red-node-watson/file/2690663401/issues/source?bid=3330694&fileBranchId=3336646#l588)